apps/home/models.py

```diff
@@ -231,6 +231,50 @@ def as_dict(self):
     def __repr__(self):
         return f'<UserFeedbacks User {self.user_id}, Stars {self.stars}>'
 
+class Namespaces (db.Model, AuditMixin):
+    __tablename__ = 'namespaces'
+    id = db.Column(db.Integer, primary_key=True)
+    description = db.Column(db.String)
+    namespace = db.Column(db.String)
+    organization = db.Column(db.String)
+    website = db.Column(db.String)
+    members = db.Column(db.String)
+    owners = db.Column(db.String)
+    is_approved = db.Column(db.Boolean, default=False)
+    is_enabled = db.Column(db.Boolean, default=False)
+    is_deleted = db.Column(db.Boolean, default=False)
+    user_id = db.Column(db.Integer, db.ForeignKey("users.id"), nullable=False)
+    user = db.relationship('Users', backref='namespaces', foreign_keys=[user_id])
```

Comment:
Can you please clarify why we need those two fields (user being a relationship)?